### PR TITLE
Feature comment edit

### PR DESCRIPTION
**Pull request by jackschultz**

### CODE REVIEW

File: app/assets/javascripts/flyover_comments/comments.coffee [hunk 1]
```diff
@@ -20,3 +20,47 @@ $ ->
...
+      $f.addClass("flyover-comment-edit-form")
+      $f.find("input[class=comment_cancel]").show()
+      textarea = $f.find("textarea")
+      textarea.attr("id", "comment_content_"+commentId)
+      textarea.text(content.text().trim())
+      content.hide()
+      container.prepend($f)
+
+  $(document).on "ajax:success", ".flyover-comment-edit-form", (e, response, status, err)->
+    console.log "Response is #{response}"
```
Comment (bcroesch) [inline app/assets/javascripts/flyover_comments/comments.coffee:52]: Delete

File: spec/features/comments_spec.rb [hunk 2]
```diff
@@ -38,6 +38,32 @@
...
+    post = FactoryGirl.create(:post)
+    other_persons_comment = FactoryGirl.create(:comment, commentable: post)
+    comment_to_edit = FactoryGirl.create(:comment, commentable: post, ident_user: current_user)
+    new_content = "Updated comment content"
+    expect(comment_to_edit.last_updated_at).to eq(nil)
+
+    visit main_app.post_path(post)
+
+    expect(page).to have_content(other_persons_comment.content)
+    expect(page).to_not have_link("edit_flyover_comment_#{other_persons_comment.id}")
```
Comment (bcroesch) [inline spec/features/comments_spec.rb:51]: Like this :+1:

**Comment by bcroesch** [comment]
@jackschultz going to merge this but delete that one line & push directly on master

File: spec/features/comments_spec.rb (hunk 2 above)
Comment (jackschultz) [inline spec/features/comments_spec.rb:51]: Ha I left this in because it was in the other test that you wrote. I see you liking your own code.

Comment (bcroesch) [inline spec/features/comments_spec.rb:51]: Awkward...

**Pull request merged**